platform.txt

```diff
@@ -99,3 +99,11 @@ tools.arduino101load.upload.params.verbose=verbose
 tools.arduino101load.upload.params.quiet=quiet
 
 tools.arduino101load.upload.pattern="{cmd.path}" "{runtime.tools.arduino101load.path}/x86/bin" {build.path}/{build.project_name}.bin {serial.port} "{upload.verbose}"
+
+# This is needed to avoid an error on unexistent fields
+tools.arduino101load.erase.params.verbose=
+tools.arduino101load.erase.params.quiet=
+tools.arduino101load.erase.pattern=
+tools.arduino101load.bootloader.params.verbose=
+tools.arduino101load.bootloader.params.quiet=
+tools.arduino101load.bootloader.pattern=/bin/bash {runtime.tools.flashpack.path}/flash_dfu.sh
```

Comment:
If you look at how we did it in Edison you should see how we used a different method for each OS